virtual_rainforest/models/abiotic/__init__.py

```diff
@@ -0,0 +1,16 @@
+import json
+from pathlib import Path
+
+from virtual_rainforest.core.config import register_schema
+
+
+@register_schema("abiotic")
+def schema() -> dict:
+    """Defines the schema that the abiotic module configuration should conform to."""
+
+    schema_file = Path(__file__).parent.resolve() / "abiotic_schema.json"
```

Comment:
This is fine for now, but note that it isn't *strictly* how you should load files embedded in a Python package. See issue #167.

It will work for now though, so don't worry about it yet

Reply:
ok, I'll keep an eye on the issue and update when we have resolved it